Makefile

```diff
@@ -41,6 +41,7 @@ linuxrelease:
 osxrelease:
 	mkdir -p release
 	cd bin && tar cvfz ../release/docker-credential-osxkeychain-v$(VERSION)-amd64.tar.gz docker-credential-osxkeychain
+	cd bin && tar cvfz ../release/docker-credential-pass-v$(VERSION)-osx-amd64.tar.gz docker-credential-pass
```

Comment:
This probably doesn't match either 🤔 (I see `osx`)

Reply:
hum indeed yes, I think `darwin` is more aligned with what we ship in other projects right?